packages/logger/tests/unit/Logger.test.ts

```diff
@@ -1227,11 +1227,10 @@ describe('Class: Logger', () => {
     test('when used as decorator on an async method, the method is awaited correctly', async () => {
 
       // Prepare
-      const injectLambdaContextAfterOrOnErrorMock = jest.fn().mockReturnValue('worked');
-      // Temporarily override the cleanup static method so that we can "spy" on it.
-      // This method is always called after the handler has returned in the decorator
-      // implementation.
-      Logger.injectLambdaContextAfterOrOnError = injectLambdaContextAfterOrOnErrorMock;
+      const injectLambdaContextAfterOrOnErrorSpy = jest.spyOn(
```

Comment:
I changed this because I realised that the previous mock was not being cleared correctly, and we really only need a spy.